hyperspy/drawing/utils.py

```diff
@@ -488,8 +490,14 @@ def plot_images(images,
                 fig=None,
                 vmin=None,
                 vmax=None,
+                overlay=False,
+                colors='auto',
+                alphas=1.0,
+                legend_picking=True,
+                legend_loc='upper right',
+                pixsize_factor = 10,
```

Comment:
Set its default value to 1 and rename to `pixel_size_factor` to avoid using abbreviation?